tests/fixtures/netexidf2ntfs/input/netex/OFFRE_INVALID/calendriers.xml

```diff
@@ -1,3 +1,905 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <PublicationDelivery xmlns="http://www.netex.org.uk/netex">
+   <dataObjects>
```

Comment:
it's a really big fixture for an invalid offer

Reply:
Right, I did copy the valid one into this `OFFRE_INVALID` folder because the file `calendriers.xml` is needed... Anyway, I've reduced the size of the file in dddf3ec8f48c3ed0a71cc362d1ccd22b7cc83af9.